src/plugins/guided_onboarding/public/components/guide_panel_flyout/index.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import {
+  EuiButtonEmpty,
+  EuiPanel,
+  EuiPortal,
+  EuiOverlayMask,
+  EuiFocusTrap,
+  EuiThemeComputed,
+} from '@elastic/eui';
+import { GuideConfig, GuideStep as GuideStepType, StepConfig } from '@kbn/guided-onboarding';
+import { i18n } from '@kbn/i18n';
+import { GuidePanelFlyoutHeader } from './guide_panel_flyout_header';
+import { GuidePanelFlyoutBody } from './guide_panel_flyout_body';
+import type { PluginState } from '../../../common';
+import { GuidePanelFlyoutFooter } from './guide_panel_flyout_footer';
+
+export const GuidePanelFlyout = ({
+  isOpen,
+  isDarkTheme,
+  toggleGuide,
+  isGuideReadyToComplete,
+  guideConfig,
+  styles,
+  navigateToLandingPage,
+  stepsCompleted,
+  pluginState,
+  handleStepButtonClick,
+  isLoading,
+  euiTheme,
+  openQuitGuideModal,
+  completeGuide,
+}: {
+  isOpen: boolean;
+  isDarkTheme: boolean;
+  toggleGuide: () => void;
+  isGuideReadyToComplete: boolean;
+  guideConfig?: GuideConfig;
+  styles: any;
```

Comment:
I think instead of `any` we could use something like `ReturnType<typeof getGuidePanelStyles>`

Reply:
Thank you, missed this. It was a placeholder